var/spack/repos/builtin/packages/py-tensorflow/package.py

```diff
@@ -647,7 +651,7 @@ def build(self, spec, prefix):
             if '~aws' in spec:
                 args.append('--config=noaws')
 
-            if '~gcp' in spec:
+            if '~gcp' in spec and not spec.satisfies('@2.0.0:2.0.1'):
```

Comment:
Can you explain this change? Is it related to https://github.com/tensorflow/tensorflow/issues/34878? As far as I can tell, 2.0.0 and 2.0.1 _should_ support the `--config=nogcp` flag. It's present in the `.bazelrc`.

Reply:
yes, it's that. I can dig further, but that trivially makes the issue go away.

Reply:
also, this was RHEL7.7/gcc7.4.0/python3.7.6

Reply:
Does it still build GCP support even when `~gcp` is requested? Another way to make the issue go away would be to use 2.1.0, or to use `~gcp` in the command line/`packages.yaml`. I don't imagine the TF devs will ever respond to my issue unfortunately.

Reply:
Well, more homework led me to what I think is the commit that "fixed" it:
```
commit 837c8b6b35429ec99d977c609fe65c0e79f4569a
Author: Gunhan Gulsoy <gunan@google.com>
Date:   Wed Sep 4 13:43:49 2019 -0700

    Remove contrib cloud bigtable and storage ops/kernels.
    Thse have moved to github.com/tensorflow/io
    PiperOrigin-RevId: 267222179
```
Will update again when I know for certain.

Reply:
Confirmed - a backport of that commit does the trick. I'll change to using that instead.